js/ui/notificationDaemon.js

```diff
@@ -825,3 +717,315 @@ const Source = new Lang.Class({
         }
     }
 });
+
+const PRIORITY_URGENCY_MAP = {
+    low: MessageTray.Urgency.LOW,
+    normal: MessageTray.Urgency.NORMAL,
+    high: MessageTray.Urgency.HIGH,
+    urgent: MessageTray.Urgency.CRITICAL
+};
+
+const GtkNotificationDaemonNotification = new Lang.Class({
+    Name: 'GtkNotificationDaemonNotification',
+    Extends: MessageTray.Notification,
+
+    _init: function(source, notification) {
+        this.parent(source);
+        this._serialized = GLib.Variant.new('a{sv}', notification);
+
+        let { "title": title,
```

Comment:
The other places in the shell are using without "". E.g. js/ui/workspace.js:770:        let { rows: rows, scale: scale } = layout;